akka-http/src/main/scala/akka/http/scaladsl/server/Directives.scala

```diff
@@ -36,7 +36,7 @@ trait Directives extends RouteConcatenation
   with SchemeDirectives
   with SecurityDirectives
   with WebSocketDirectives
-  with ReverseProxyDirectives
   with FramedEntityStreamingDirectives
+  with ReverseProxyDirectives
```

Comment:
ok, this is not bin-compat but that's fine _currently_, and we'll deprecate the trait mixin style in the new repo (`trait Directives`) so this should be compatible from nowon anyway, good